src/main/java/uk/gov/hmcts/reform/pip/model/enums/Platform.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.hmcts.reform.pip.model.enums;
+
+/**
+ * Enum for platforms.
+ */
+public enum Platform {
+
+    CFT, CP
```

Comment:
ooo thast my bad, `platform` here is meant to refer to `hearing platform` so like skype or Teams etc

Reply:
I've double checked on this one and I believe the above list to be the right ones for hearing platform however there may end up being more added, therefore I've I've left this as it is for now until the complete list is available